src/python/platforms/fuchsia/util/device.py

```diff
@@ -231,9 +231,9 @@ def _guess_pid(self):
         """
     out = self._dump_log(['--only', 'reset,Fuzzer,Sanitizer'])
     pid = -1
-    for line in out.split('\n'):
+    for line in out.split(b'\n'):
```

Comment:
why not decode here ?

Reply:
Seems simpler to work on bytes here, as this is just a few lines and the resulting bytes isn't propagated to anywhere.